src/main/scala/org/zalando/spearheads/innkeeper/dao/PathsPostgresRepo.scala

```diff
@@ -73,15 +73,37 @@ class PathsPostgresRepo @Inject() (
     }
   }
 
-  override def pathWithUriHostIdExists(uri: String, hostIds: Seq[Long]): Future[Boolean] = {
-    logger.debug("uri hostId pairs are duplicate check")
+  override def collisionExistsForPath(path: PathIn): Future[Boolean] = {
+    logger.debug("path uri collision check")
+
+    def position = SimpleExpression.binary[String, String, Int] {
+      (subString, string, queryBuilder) =>
+        import slick.util.MacroSupport._
```

Comment:
I wouldn't use the inner import feature of scala... I would move the imports to the beginning of the file... imports are details which should be managed by the IDEs...